src/tests/multihost/alltests/test_default_debug_level.py

```diff
@@ -204,3 +213,156 @@ def test_0005_bz1915319(self, multihost, backupsssdconf):
+        find_logs(multihost, "/tmp/grp_file", "Group111")
+        find_logs(multihost, "/tmp/grp_file", "Group22")
+
+    @staticmethod
+    def test_bz785898(multihost, backupsssdconf):
+        """
+        :title: Enable midway cache refresh by default bz785898
+        :id: 39695a7a-a29c-11ed-b2b1-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=785898
+        :steps:
+            1. Set entry_cache_timeout = 10. Leave
+                entry_cache_nowait_percentage unset (at defaults).
+            2. Run 'time getent passwd <legitimate_user>'
+                30 times, with a 3-second sleep.
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+        """
+        client = multihost.client[0]
+        tools = sssdTools(multihost.client[0])
+        ldap_uri = f'ldap://{multihost.master[0].sys_hostname}'
+        ds_rootdn = 'cn=Directory Manager'
+        ds_rootpw = 'Secret123'
+        ldap_inst = LdapOperations(ldap_uri, ds_rootdn, ds_rootpw)
+        user_dn = f'uid=mid_cacheuser,{ds_suffix}'
+        user_info = {'cn': 'Midcache'.encode('utf-8'),
+                     'loginshell': '/bin/bash'.encode('utf-8'),
+                     'uidNumber': '121298'.encode('utf-8'),
+                     'gidNumber': '10000'.encode('utf-8'),
+                     'objectClass': ['top'.encode('utf-8'),
+                                     'posixAccount'.encode('utf-8')],
+                     'homeDirectory': '/home/mid_cacheuser'.encode('utf-8'),
+                     'userPassword': 'Secret123'.encode('utf-8')}
+        ldap_inst.add_entry(user_info, user_dn)
+        tools.sssd_conf("domain/example1",
+                        {'entry_cache_timeout': '60',
+                         'cache_credentials': True,
+                         'use_fully_qualified_names': False}, action='update')
+        tools.sssd_conf("nss", {'filter_groups': 'root',
+                                'filter_users': 'root',
+                                'debug_level': '9',
+                                'entry_cache_nowait_percentage': '50',
+                                'memcache_timeout': '1'}, action='update')
+        tools.sssd_conf("sssd", {'sbus_timeout': '30'}, action='update')
+        tools.clear_sssd_cache()
+        client.run_command("mkdir /tmp/lookup")
+        for i in range(30):
+            client.run_command("> /var/log/sssd/sssd_example1.log")
+            client.run_command("getent passwd -s sss mid_cacheuser")
+            time.sleep(3)
+            client.run_command(f"cat /var/log/sssd/sssd_example1.log > /tmp/lookup/lookup{i}")
+        client.run_command("grep -r 'Got request for' /tmp/lookup/")
+        client.run_command("grep -r 'Got request for' /tmp/lookup > /tmp/lookupfile")
+        time.sleep(3)
+        ldap_inst.del_dn(f"uid=mid_cacheuser,{ds_suffix}")
+        assert int(client.run_command("cat /tmp/lookupfile | wc -l").stdout_text.split()[0]) > 2
```

Comment:
You can just get the file, split it by lines and get length of the list thi list instead of bash pipelines.